NEXT_CHANGES.rst

```diff
@@ -9,5 +9,208 @@ links to relevant Issues, Discussions, and PR's on github with the following for
+  of sensitivity factors (themselves saveable as ``ms_calculators.MSCalResult``s).
+  - available_series_names: "n_dot_{mol}" for all the mol in its ``mol_list``
+  Constructor methods:
+  - ``gas_flux_calibration``, formerly a method of ``MSMeasurement``.
+  - ``gas_flux_calibration_curve``, formerly a method of ``MSMeasurement``.
+
+- ``ms_calculators.MSBackgroundSet`` is a new calculator that replaces the poor
+  incomplete implementation of backgrounds previously in ``MSCalibration``. The
+  structure is similar to ``MSCalibration`` in that a ``MSBackgroundSet`` contains
+  a set of saveable ``MSBackground`` objects, each for a single m/z. So far, only
+  one type of background is implemented, the ``MSConstantBackground``.
+  - available_series_names: mass_list.
+  - Note that The available series names have the same names as the corresponding
+    raw data series before background subtraction. To get the raw series, grab or
+    look up ``f"{mass}-raw"`` or ``grab`` with ``remove_background=False``.
+
+- ``ecms_calculators.ECMSCalibration`` is not a real calculator in the sense that
+  it doesn't do *quantification*. Instead, it does *calibrtion*, and its calibration
+  methods, listed below, all return ``MSCalibration`` objects. This is consistent
+  with the fact that an EC-MS calibration experiment can be used to obtain sensitivity
+  factors for a setup which is then used without electrochemistry (e.g. for thermal
+  catalysis measurements). The calibration methods are
+  - ``ecms_calibration``, formerly a method of ``ECMSMeasurement``
+  - ``ecms_calibration_curve``, also formerly a method of ``ECMSMeasurement``
+
+- ``ecms_calculators.ECMSImpulseResponse``, moved from **deconvolution.py**, is the
+  deconvolution calculator. An ``ECMSImpulseResponse`` object (``imp_resp``)
+  describes the response of one ``mol``. It's demonstrated in
+  **deconvolution_demo.py**.
+  - available_series_names: "n_dot_{mol}-deconvoluted". Getting this from an
+    ``MSMeasurement`` with ``imp_resp`` attached requires that there is also another
+    calculator which provides "n_dot_{mol}".
+  Constructor methods:
+  - ``from_measurement`` takes the shape of the impulse response from a measurement
+    representing an impulse experiment, i.e. one where a short burst of product (e.g.
+    "H2" form hydrogen evolution) is produced at the electrode and its mass signal,
+    after broadening by mass transport between the electrode and the inlet, recorded.
+  - ``from_model`` calculates the shape of the impulse response according to a mass
+    transport model.
+
+- ``xrf_calculators.TRXRFCalculator`` is a built-in calculator of ``TRXRFMeasurement``
+  which replaces a simple series_constructor.
+  - available_series_names: "FF_over_I0"
+
+
+One ``Calculator``, the ``siqCalculator`` for advanced MS and EC-MS calibration,
+is implemented as a plugin. At present, this is in **src/ixdat/config.py** but it
```

Comment:
Good point!
Okay, I played around and got it to work with a new folder called "plugs" (it would be called "plugins" but then it would be confusing, since the object of `PluginOptions` is called plugins).
So now this class is in **ixdat/plugs/siq_plugin.py**.